packages/amplify_auth_cognito/lib/method_channel_auth_cognito.dart

```diff
@@ -275,26 +275,26 @@ class AmplifyAuthCognitoMethodChannel extends AmplifyAuthCognito {
 
   @override
   Future<UpdateUserAttributeResult> updateUserAttribute(
-      {@required UpdateUserAttributeRequest request}) async {
+      {UpdateUserAttributeRequest? request}) async {
```

Comment:
Similar to above, I think this should be required. Not sure if I am missing something.

Reply:
Same as above.